docs/manage/remotes.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "Parts, Sub-Parts and Remotes"
+linkTitle: "Remotes and Sub-Parts"
+weight: 40
+type: "docs"
+description: "Connect robots to each other."
+tags: ["server", "components", "services"]
+---
+
+When robots can communicate with each other, they can share resources and operate collaboratively.
+This document explains how to establish secure connections between robots.
+
+### Robot parts
+
+Robots are organized into *parts*, where each part represents a computer (a single-board computer like a Raspberry Pi or a desktop, laptop, or other computer) running `viam-server`, the hardware [components](/components/) attached to it, and any [services](/services/) or other resources running on it.
+Every robot has a main part.
+Multi-part robots also have one or more *sub-parts* representing additional computers running `viam-server`.
+
+There are two ways to link robot parts:
+
+- **Sub-part**: If you have two computers within the *same robot*, use one as the main part and [connect the other to it as a sub-part](#configure-a-sub-part).
+
+- **Remote**: To connect two computers that are parts of *different robots*, [add one robot part as a remote part of the other robot](#configure-a-remote).
+
+Connections between robots are established using the best network path available.
+
+When you configure a remote or a sub-part, the main robot part can access all the components and services configured on the remote robot part as though they were entities of the main robot part.
+This is a one-way connection: The main robot part can access the resources of the remote robot part, but the remote robot cannot access the resources of the robot part remoting into it.
+
+<img src="../img/remotes/remotes-diagram.png" alt="Example of a remote and a two part robot where the main (and only) part of robot 1 remotes into the main part of robot 2, and thus has access to all resources of robot 2." >
+
+## Configuration
+
+### Configure a sub-part
+
+You can make a multi-part robot by first configuring one part which is the "main" part, and then configuring one or more sub-parts.
+The main part will be able to access the resources of its sub-parts.
+Sub-parts will *not* have access to the resources of the main part.
+
+Use the parts drop-down menu in the top banner of your robot's page on the [Viam app](https://app.viam.com) to add a new sub-part:
+
+![The Viam app interface with the part drop-down open. A new part called "my-sub-part" is being created.](../img/remotes/sub-part-config.png)
+
+<br>
+
+{{% alert title="Note" color="note" %}}
+
+When you create a sub-part, a connection is established between the main part and the sub-part without you needing to explicitly configure a remote.

```

Comment:
TBH I am not sure if this is still needed.  It feels already clear-cut to me.

Reply:
Yeah fair enough but idk whether to remove it. Do you think it hurts anything?

Reply:
yeah I think we can remove it - will add the suggestion on the other PR

Reply:
Removed on the other PR